README.md

```diff
@@ -30,165 +32,471 @@ The two guiding principles are:
+
+## Configuration
 
-```bash
-$ yarn add hops-react@next react-dom react-helmet
+There are two different means of configuring your application:
+
+- there are [**"settings"**](#settings) that you configure at build time, these are provided through the `package.json` or a separate file.
+- and there are [**"options"**](#options) that you pass in the second argument to the `render()` function.
+
+### Settings
+
+You can provide settings to a Hops application via either:
+
+- a `"hops"` key in your `package.json`
+- a `hops.config.js` file in your root directory that exports an object containing all your settings.
+- a `.hopsrc{.json,.yaml,.js}` file in your root directory.
+
+_For simplicity we use the first variant in all examples._
+
+#### Default settings
+
+| Name        | Type                 | Default          | Example                                                            | Description                                                                                  |
+| ----------- | -------------------- | ---------------- | ------------------------------------------------------------------ | -------------------------------------------------------------------------------------------- |
+| `https`     | `Boolean` / `Object` | `false`          | `true` or<br/>`{ "keyFile": "./my.key", "certFile": "./my.cert" }` | Configure HTTPS support for Hops                                                             |
+| `host`      | `String`             | `[HOST]`         | `10.10.10.10`                                                      | Specify the IP address that Hops should bind to                                              |
+| `port`      | `String`             | `[PORT]`         | `1337`                                                             | Specify the Port that Hops should listen on                                                  |
+| `locations` | `Array<String>`      | `[]`             | `["/", "/about"]`                                                  | An array of locations for static rendering of HTML pages                                     |
+| `basePath`  | `String`             | `''`             | `/my-app`                                                          | The URL base path from which your application will be served                                 |
+| `assetPath` | `String`             | `<basePath>`     | `<basePath>/assets`                                                | The URL base path from which the assets will be served from                                  |
+| `buildDir`  | `String`             | `<rootDir>/dist` | `<rootDir>/out`                                                    | The directory in which the build outputs will be written to                                  |
+| `browsers`  | `Array<String>`      | `['defaults']`   | [`last 1 Chrome versions`]                                         | An array of browserslist queries to specify targets for which to transpile/polyfill.         |
```

Comment:
```diff
- [`last 1 Chrome versions`]
+ `['last 1 Chrome versions']`
```

… also we should link to the babel-preset-env documentation here.

Reply:
done